### ISSUE TITLE
Compensation early slashes

### ISSUE THREAD

**Issue by DavyChang** [Phala-Network/phala-blockchain#719]
When Khala went online last year, there should be no slash for mining, but due to some mistakes, some miners were indeed slashed, and they should be compensated.

example
PID: 203
Delegator Address: 41r5zbukjR3c51Mk2sYZ2bjdRWoTSrSrptn3tjDmcXmiQyQc

**Comment by h4x3rotab** [comment]
Agree. The reason is that at the launch of the network, we accidentally loads Phala tokenomic parameters with slash enabled. This doesn't match the launch proposal in which the slash should be disabled. Once we noticed it, we have disabled the slash, and also updated Gatekeeper to fix all the miners with `V < Ve`. The `V` of these miners have been reset to their `Ve`, and thus there will not trigger any actual slash on the stake. However, before the fix, some miners are already stopped. If their `V` is less than `Ve`, the stake for the miners are actually slashed. This is unexpected, and it's reasonable to compensate the affected users.

To do so, we need to scan all the slash event before the V fix, and figure out a list of the affected workers.
